src/sass/base/_b-utils.scss

```diff
@@ -12,11 +12,11 @@
   border: 0;
   border-radius: 0;
   outline: 0;
-  padding: 0;
+  padding: 0 !important;
```

Comment:
Would it be helpful to use comments to explain why !important is being used here?